riot/riot.py

```diff
@@ -158,14 +162,28 @@ class Session:
     @classmethod
     def from_config_file(cls, path: str) -> "Session":
         spec = importlib.util.spec_from_file_location("riotfile", path)
+        if not spec:
+            click.echo(
+                f"Invalid file format for riotfile. Expected file with .py extension got '{path}'.",
+                err=True,
+            )
+            sys.exit(1)
```

Comment:
This is class code, it'd be better not to have `sys.exit` nor `click` code here :(

Reply:
True, it felt kinda icky to me too. I'll see if we can work it into the click code instead and use normal exception handling